packages/api/src/utils/codecs.ts

```diff
@@ -20,11 +19,8 @@ export type EmptyRequest = Record<never, never>;
 export type EmptyResponseData = void;
 export type EmptyMeta = void;
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
`any` is allowed now?

Reply:
No, this rule is disabled for all files in `packages/api/src/utils/**/*.ts`, as there are a lot of occurrences to disable per line.

Reply:
I didn't went through line by line manually for such rules, searched through and if there I found more occurrences of disabled rules in single file added those as override. Considering that all those overrides will be removed very soon.